feat: add OpenRPC spec (#2801)
massalabs/massa: docs/technical-doc/api.rst

    curl -X POST -H "Content-Type: application/json" -d '{"jsonrpc": "2.0", "method": "node_stop", "id": 123 }' 127.0.0.1:33034

> You could interact with Massa API on `OpenRPC Playground <https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/massalabs/massa/main/docs/technical-doc/openrpc.json>`_.

Reviewer: ```suggestion
You can interact with the Massa RPC on `OpenRPC Playground <https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/massalabs/massa/main/docs/technical-doc/openrpc.json>`_.
```
